src/firefly/js/ui/FileUpload.jsx

```diff
@@ -139,7 +163,7 @@ function handleChange(ev, fireValueChange, type, fileAnalysis) {
     });
 }
 
-function makeDoUpload(file, type, isFromURL, fileAnalysis) {
+function makeDoUpload(file, type, isFromURL, isFromWsURL, fileAnalysis) {
```

Comment:
parameter 'isFromWsURL' is introduced into this function 'makeDoUpload', please check if every function call on 'makeDoUpload' in this file has add this newly added parameter.